scripts/e2e-deploy.sh

```diff
@@ -84,16 +84,10 @@ DEPLOY_PROJECT=hccm
 # location of commands
 OC=$(which oc)
 OCDEPLOYER=$(which ocdeployer)
-IQE=$(which iqe)
-
-if [[ "$OSTYPE" == "darwin"* ]]; then
-    KOKU_SECRETS=$PWD/e2e-secrets.yml
-else
-    KOKU_SECRETS=$(dirname $(readlink -f $0))/e2e-secrets.yml
-fi
-
-UDT_SRR="$(dirname ${KOKU_SECRETS})/update_e2e_srr.py"
-
+SCRIPT_DIR="$( pushd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+IQE_CONTAINER="$SCRIPT_DIR/../testing/run_test.sh"
+KOKU_SECRETS="$SCRIPT_DIR/e2e-secrets.yml"
+UDT_SRR="$SCRIPT_DIR/update_e2e_srr.py"
```

Comment:
This is two things:

1. changing to use the iqe container instead of a local install.
2. I found a way to identify the script path that works cross-platform, eliminating the need for the OS-specific if-then.